cmd/describe/machinepool/machinepool.go

```diff
@@ -0,0 +1,59 @@
+package machinepool
+
+import (
+	"fmt"
+
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	ocmOutput "github.com/openshift/rosa/pkg/ocm/output"
+	"github.com/openshift/rosa/pkg/output"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+func describeMachinePool(r *rosa.Runtime, cluster *cmv1.Cluster, clusterKey string, machinePoolID string) error {
+	r.Reporter.Debugf("Fetching machine pool '%s' for cluster '%s'", machinePoolID, clusterKey)
+	machinePool, exists, err := r.OCMClient.GetMachinePool(cluster.ID(), machinePoolID)
+	if err != nil {
+		return err
+	}
+	if !exists {
+		r.Reporter.Infof("Machine pool '%s' not found", machinePoolID)
```

Comment:
I wonder if we should error out instead.
From the describe cluster command:
```
oadler@fedora:rosa (master)$ rosa describe cluster --cluster oadler
E: Failed to get cluster 'oadler': There is no cluster with identifier or name 'oadler'
```
WDYT?

Reply:
I think we are not totally consistent, but yes let's converge on returning an error and exit code 1 for not found case.
example
```
rosa describe machinepool -c igal-rosa-test6 --machinepool workers
E: Machine pool 'workers' not found
```